pkg/csi_driver/controller.go

```diff
@@ -76,11 +76,13 @@ const (
 	// User provided labels
 	ParameterKeyLabels = "labels"
 
-	// Keys for tags to attach to the provisioned disk.
+	// Keys for tags to attach to the provisioned Filestore shares and instances.
 	tagKeyCreatedForClaimNamespace = "kubernetes_io_created-for_pvc_namespace"
 	tagKeyCreatedForClaimName      = "kubernetes_io_created-for_pvc_name"
 	tagKeyCreatedForVolumeName     = "kubernetes_io_created-for_pv_name"
 	tagKeyCreatedBy                = "storage_gke_io_created-by"
+	tagKeyClusterName              = "gke_cluster_name"
```

Comment:
maybe storage_gke_io_cluster_name and storage_gke_io_cluster_location ?

Reply:
the reason I didn't use `storage_gke_io_` prefix is because that might be too similar with the `storage_gke_io_created-by` and since we expect customers to change the cluster name and location tag I think simpler, easier to read tag keys make more sense

Reply:
I think the primary indicator of the tag prefix is storage_gke_io_* are used by CSI driver to manage stuff. User are free to modify labels anyway. @mattcary  what are your thoughts on the tag naming?

Reply:
We should be consistent with the pd driver, which uses [storage.gke.io/created-by](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/pkg/common/parameters.go#L48) if I read the code correctly.

That's the principle of least surprise at least.

Reply:
I think the `tagCreatedBy` is inconsistent because [filestore tag key constraint](https://cloud.google.com/filestore/docs/managing-labels#requirements):
> Keys and values can contain only lowercase letters, numeric characters, underscores, and dashes

as for the cluster name and location I think having multiple tag key start with `storage_gke_io_ might be confusing: https://screenshot.googleplex.com/Auh94HCt9cHnTFR

Reply:
I think we have to have a google-specific prefix, though, or we're going to clash with customer-provided tags. So no matter we do I think the current UI isn't going to be great and we may need to fix that.

We have an open issue to start putting the PD tags into labels as well as the description, so the encoding issue is going to come up again (labels didn't exist I guess when the csi driver was first made).

I think it's important to have uniformity across our products. And in that context the names always end up a bit clunky.

Reply:
Agree. UI should not drive the label names, if should be the other way around. For uniformity, and clear identification of GKE managed tags, let's proceed with storage_gke_io_ prefixed label for the cluster name. storage_gke_io_cluster_name for example

Reply:
sgtm done.